src/lib/classes/universal-profile.spec.ts

```diff
@@ -619,11 +622,11 @@ describe('UniversalProfile', () => {
           signers[0]
         );
 
-        const data = await universalProfile.getDataBatch([
-          LSP3_UP_KEYS.UNIVERSAL_RECEIVER_DELEGATE_KEY,
-        ]);
+        const data = await universalProfile.getData(
```

Comment:
Maybe we could rename data to something more description like urdAddress?

Reply:
```suggestion
        const univeralReceiverDelegateAddress = await universalProfile.getData(
```

Reply:
Definitely